bots/image-download

```diff
@@ -224,12 +260,16 @@ def prune_images(force, dryrun):
     # iterate over all visible refs (mostly branches)
     # this hinges on being in the top level directory of the the git checkout
     with remember_cwd():
-        os.chdir(os.path.join(BASE, ".."))
+        os.chdir(os.path.join(BOTS, ".."))
         refs = get_refs()
```

Comment:
Not sure what this means. I'll assume it's just thinking out loud. The pruning needs to move to its own file/task anyway.